channeldb/kvdb/etcd/db.go

```diff
@@ -59,3 +105,6 @@ func (db *db) Copy(w io.Writer) error {
 func (db *db) Close() error {
 	return db.cli.Close()
 }
+
+// TODO
+//func (db *db) Batch(apply func(tx kvdb.RwTx) error) error { }
```

Comment:
Batch can just call `Update`, since `etcd` as is, is inherently multi-writer.

Reply:
So do I understand correctly that `testBatchInterface` in walletdb is incorrect as it calls `CreateTopLevelBucket` in each concurrent batch? As that'll make the first batch committed to succeed the rest to fail. It's simple to correct that test if what I assume is right and in that case `Batch` is indeed a simple `Update` call.

Reply:
depends on https://github.com/btcsuite/btcwallet/pull/679